test/script/RebalanceScUsdcV2.s.t.sol

```diff
@@ -0,0 +1,532 @@
+        uint256 expectedDebt = priceConverter.usdcToEth(expectedCollateral).mulWadDown(script.morphoTargetLtv());
+
+        script.run();
+
+        assertApproxEqRel(vault.wethInvested(), expectedDebt, 0.001e18, "weth invested");
+        assertApproxEqRel(vault.totalDebt(), expectedDebt, 0.001e18, "total debt");
+        assertApproxEqRel(vault.totalCollateral(), expectedCollateral, 0.001e18, "total collateral");
+        assertApproxEqRel(vault.usdcBalance(), expectedFloat, 0.001e18, "usdc balance");
+    }
+
+    function test_run_twoAdaptersInitialRebalance() public {
+        uint256 morphoInvestableAmountPercent = 0.4e18;
+        uint256 aaveV2InvestableAmountPercent = 0.6e18;
+        script.setMorphoInvestableAmountPercent(morphoInvestableAmountPercent);
+        script.setAaveV2InvestableAmountPercent(aaveV2InvestableAmountPercent);
+
+        uint256 expectedFloat = vault.totalAssets().mulWadDown(vault.floatPercentage());
+        uint256 expectedCollateral = vault.usdcBalance() - expectedFloat;
+        uint256 expectedDebt = priceConverter.usdcToEth(expectedCollateral).mulWadDown(script.morphoTargetLtv());
+
+        script.run();
+
+        assertApproxEqRel(vault.wethInvested(), expectedDebt, 0.001e18, "weth invested");
+        assertApproxEqRel(vault.totalDebt(), expectedDebt, 0.001e18, "total debt");
+        assertApproxEqRel(vault.totalCollateral(), expectedCollateral, 0.001e18, "total collateral");
+        assertApproxEqRel(vault.usdcBalance(), expectedFloat, 0.001e18, "usdc balance");
+
+        uint256 morphoExpectedCollateral = expectedCollateral.mulWadDown(morphoInvestableAmountPercent);
+        uint256 aaveV2ExpectedCollateral = expectedCollateral.mulWadDown(aaveV2InvestableAmountPercent);
+        uint256 morphoExpectedDebt =
+            priceConverter.usdcToEth(morphoExpectedCollateral).mulWadDown(script.morphoTargetLtv());
+        uint256 aaveV2ExpectedDebt =
+            priceConverter.usdcToEth(aaveV2ExpectedCollateral).mulWadDown(script.aaveV2TargetLtv());
+
+        assertApproxEqRel(vault.getCollateral(morpho.id()), morphoExpectedCollateral, 0.001e18, "morpho collateral");
+        assertApproxEqRel(vault.getCollateral(aaveV2.id()), aaveV2ExpectedCollateral, 0.001e18, "aave v2 collateral");
+        assertApproxEqRel(vault.getDebt(morpho.id()), morphoExpectedDebt, 0.001e18, "morpho debt");
+        assertApproxEqRel(vault.getDebt(aaveV2.id()), aaveV2ExpectedDebt, 0.001e18, "aave v2 debt");
+    }
+
+    function test_run_leverageDownByRepayingDebt() public {
+        script.run(); // initial rebalance
+        script = new RebalanceScUsdcV2TestHarness(); // reset script state
+
+        assertTrue(vault.wethInvested() > 0, "weth invested");
+        assertTrue(vault.totalDebt() > 0, "total debt");
+        assertTrue(vault.totalCollateral() > 0, "total collateral");
+        assertTrue(vault.usdcBalance() > 0, "usdc balance");
+
+        script.setMorphoTargetLtv(script.morphoTargetLtv() - 0.1e18);
```

Comment:
There should be an assertion in this test right, to check if the new target ltv of morpho has been reached?

Reply:
[fixed](https://github.com/lindy-labs/sandclock-contracts/pull/118/commits/8fde687c2d19830a3d84d5b551bf418df4e0bd20)